src/TaggerCommand.php

```diff
@@ -97,17 +104,4 @@ protected function execute(InputInterface $input, OutputInterface $output) : int
 
         return 0;
     }
-
-    private function tagWithChangelog(string $tagName, string $package, string $version, string $changelog) : bool
```

Comment:
I think the tagging should likely be the same between different providers; what will differ is _releasing_. We may also want to vary what message is generated for the tag, as I noted earlier.

As such, I'd argue that the interface should remove the `createLocalTag()` method, and that should move back to this class. We can then vary the message created, using either `--no-tag-summary` or `--short` or similar.